docs/development.md

```diff
@@ -175,3 +177,53 @@ helm get manifest akri | less
 
 ### Helm Upgrade
 To modify an Akri installation to reflect a new state, you can use [`helm upgrade`](https://helm.sh/docs/helm/helm_upgrade/). See the [Customizing an Akri Installation document](./customizing-akri-installation.md) for further explanation. 
+
+## Naming Guidelines
+
+One of the [two hard things](https://martinfowler.com/bliki/TwoHardThings.html) in Compute Science is naming things. It is proposed that Akri adopt naming guidelines to make developers' lives easier by providing consistency and reduce naming complexity.
+
+Akri existed before naming guidelines were documented and may not employ the guidelines summarized here. However, it is hoped that developers will, at least, consider these guidelines when extending Akri.
+
+### General Principles
+
++ Akri uses English
++ Akri is written principally in Rust and Rust [Naming](https://rust-lang.github.io/api-guidelines/naming.html) conventions are used
```

Comment:
nit comma and cap: "in Rust , and Rust naming..."

Reply:
Fixed: https://github.com/deislabs/akri/pull/204/commits/9fa40708c0d600eb602ecf232f0cca7a680c0e7d